target/linux/lantiq/image/Makefile

```diff
@@ -643,23 +688,30 @@ endef
 
 define Device/avm_fritz3370-rev2-hynix
   $(Device/avm_fritz3370)
+  DEVICE_VENDOR := AVM
+  DEVICE_MODEL := FRITZ!Box 3370
+  DEVICE_VARIANT := Rev. 2
```

Comment:
You will inherit vendor and model from fritz3370. Just override VARIANT in this node. Consider introducing a dinstinctive name for plain Rev. 2.

Reply:
I would go with "2".

Reply:
"Rev. 2" is probably on the box. Let's leave it.